models/lfs_lock.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	api "code.gitea.io/sdk/gitea"
+)
+
+// LFSLock represents a git lfs lock of repository.
+type LFSLock struct {
+	ID      int64 `xorm:"pk autoincr"`
+	RepoID  int64 `xorm:"INDEX"`
+	Owner   *User `xorm:"-"`
+	OwnerID int64 `xorm:"INDEX"`
+	Path    string
```

Comment:
Path with `varchar(255)` if not xorm tag here. Is that enough length?

Reply:
Good catch switch to TEXT and add a corresponding test

Reply:
Maybe `xorm:"varchar(1024)"` or `xorm:"varchar(2048)"` is enough.